AfricanReligions/localisation/afr_event_texts.csv

```diff
@@ -2,8 +2,8 @@ EVTOPTEHF23004;The myths of the Dark Continent intrigue me.;;;;;;;;;;;;;x
 EVTTITLE_AFRREL003;The Inna's Apotheosis;;;;;;;;;;;;;x
 EVTDESC_AFRREL003;The Inna, the mother of us all, for long has being reduced to the role of a mere subject or adviser of powerful rulers.\nBut those dark days are over today. All the great Magadjas have met today in the court of [From.GetTitledName] and have invoked the bori spirits to elevate [Root.RelHead.GetTitledName] to a divine character.\n\nThe Inna, possessed by the spirit of Magiro, the oldest and wisest of the bori, has received the authority to decide on all the spiritual issues that concern the faithful. The drums and marimbas rumble through the streets while the great mother is carried by her parishioners to the Kano temple, from where she will now rule the city until the end of time.;;;;;;;;;;;;x
 EVTOPTA_AFRREL003;Glory to our mother, [Root.RelHead.GetBestName]!;;;;;;;;;;;;;x
-EVTDESC_AFRREL004;To all my faithful,\nFor long the unbelievers have lived in our lands, shared our tables and enjoyed the protection of our spears. This is an example of the infinite tolerance of our faith. However it�s time for these fools who reject the truth of the Jan Gari to repay our generosity. For this reason I authorize the lords to tax their unfaithful peasantry.\nLovingly, your Inna;;;;;;;;;;;;;x
-EVTOPTA_AFRREL004;The Inna's word bring joy to my soul� and my pocket.;;;;;;;;;;;;;x
+EVTDESC_AFRREL004;To all my faithful,\nFor long the unbelievers have lived in our lands, shared our tables and enjoyed the protection of our spears. This is an example of the infinite tolerance of our faith. However it’s time for these fools who reject the truth of the Jan Gari to repay our generosity. For this reason I authorize the lords to tax their unfaithful peasantry.\nLovingly, your Inna;;;;;;;;;;;;;x
```

Comment:
You've messed up the encoding again.  You need to figure out how to make your editor only save files in Windows-1252.